.circleci/config.yml

```diff
@@ -249,7 +249,8 @@ jobs:
           command: cargo install cargo-audit
       - run:
           name: Check for known security issues in dependencies
-          command: cargo audit
+          # TODO remove ignore when https://github.com/filecoin-project/rust-fil-proofs/issues/1403 resolved
```

Comment:
Can you open an issue for this real quick if were gunna ignore? Def gunna forget this 😂